src/compiler/index.js

```diff
@@ -20,36 +23,38 @@ class Compiler {
     this.blockIpfsMethods = options.blockIpfsMethods
     this.libsDirs = []
 
-    for (
-      let dir = path.resolve(this.sourceDir);
-      // Terminate after the root dir or when we have found node_modules
-      dir !== undefined;
-      // Continue with the parent directory, terminate after the root dir
-      dir = path.dirname(dir) === dir ? undefined : path.dirname(dir)
-    ) {
-      if (fs.existsSync(path.join(dir, 'node_modules'))) {
-        this.libsDirs.push(path.join(dir, 'node_modules'))
+    if (options.protocol.name !== 'substreams') {
```

Comment:
@dotansimha This was checking `graph-ts` installation, no need for `graph-ts` in Substreams, so removed this check if protocol if `substreams`.